presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergAbstractMetadata.java

```diff
@@ -612,6 +641,12 @@ protected ImmutableMap<String, Object> createMetadataProperties(Table icebergTab
         properties.put(METADATA_DELETE_AFTER_COMMIT, IcebergUtil.isMetadataDeleteAfterCommit(icebergTable));
         properties.put(METRICS_MAX_INFERRED_COLUMN, IcebergUtil.getMetricsMaxInferredColumn(icebergTable));
 
+        SortOrder sortOrder = icebergTable.sortOrder();
+        // TODO: Support sort column transforms (https://github.com/prestodb/presto/issues/24250)
+        if (sortOrder.isSorted() && sortOrder.fields().stream().allMatch(sortField -> sortField.transform().isIdentity())) {
+            List<String> sortColumnNames = toSortFields(sortOrder);
+            properties.put(SORTED_BY_PROPERTY, sortColumnNames);
+        }
```

Comment:
Is it appropriate to throw an exception directly if there exists non-identity sorting columns?

I see in other places, such as `getSupportedSortFields(...)`, we will filter out the sorting column with identity transform. Considering the tables created in an external engine and registered into presto, it seems more reasonable. Should we unify their behavior? Any misunderstanding please let me know.